src/main/java/frc/robot/RobotContainer.java

```diff
@@ -173,9 +172,14 @@ private void configureButtonBindings() {
         operator.rightBumper().onTrue(new FlashingLEDColor(leds, Color.kPurple)
             .until(() -> this.s_wristIntake.getCubeSensor()).withTimeout(15.0));
 
-        operator.a().whileTrue(new WristIntakeIn(s_wristIntake));
-        operator.b().whileTrue(new WristIntakeRelease(s_wristIntake));
-        operator.x().whileTrue(new WristIntakeIn(s_wristIntake).alongWith(new ArmIntake(s_Arm)));
+        operator.a()
+            .whileTrue(new InstantCommand(() -> s_wristIntake.stopHoldingPiece())
+                .andThen(new WristIntakeIn(s_wristIntake)
+                    .andThen(new InstantCommand(() -> s_wristIntake.holdPiece()))));
+        operator.b().whileTrue(new WristIntakeRelease(s_wristIntake)
+            .andThen(new InstantCommand(() -> s_wristIntake.stopHoldingPiece())));
+        operator.x().whileTrue(new InstantCommand(() -> s_wristIntake.stopHoldingPiece())
+            .andThen(new ArmIntake(s_Arm).andThen(() -> s_wristIntake.holdPiece())));
```

Comment:
U may want to convert these to their own command files if the sequences need to be reused for auto or other places.

Reply:
Added into the intake commands